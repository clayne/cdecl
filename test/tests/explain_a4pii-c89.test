cdecl @ @ -xc89 @ explain *a[4] @ 0
