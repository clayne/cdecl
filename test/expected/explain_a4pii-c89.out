declare a as array 4 of pointer to int
